Can't build on 10.9.5

/usr/local/Library/Homebrew/shims/super/clang++    -std=c++11 -fno-exceptions -fPIC -Wall -Werror -pedantic -Wextra -Wno-unused-private-field -fno-exceptions -DNDEBUG -s -Wl,-search_paths_first -Wl,-headerpad_max_install_names  CMakeFiles/runCoreUnitTests.dir/RunTests.cpp.o CMakeFiles/runCoreUnitTests.dir/aws/RegionMapperTest.cpp.o CMakeFiles/runCoreUnitTests.dir/aws/auth/AWSCredentialsProviderTest.cpp.o CMakeFiles/runCoreUnitTests.dir/aws/client/AWSClientTest.cpp.o CMakeFiles/runCoreUnitTests.dir/aws/client/AWSErrorMashallerTest.cpp.o CMakeFiles/runCoreUnitTests.dir/aws/client/HttpSchemeMapperTest.cpp.o CMakeFiles/runCoreUnitTests.dir/http/HttpTypesMapperTest.cpp.o CMakeFiles/runCoreUnitTests.dir/http/URITest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/DateTimeTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/EnumOverflowTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/FileSystemUtilsTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/GetTheLightsTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/HashingUtilsTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/OSVersionInfoTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/StringUtilsTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/crypto/CryptoStreamsTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/crypto/SymmetricCiphersTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/json/JsonSerializerTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/xml/XmlSerializerTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/logging/LoggingTest.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/memory/MemoryTests.cpp.o CMakeFiles/runCoreUnitTests.dir/utils/ratelimiter/RateLimiterTests.cpp.o  -o runCoreUnitTests  ../aws-cpp-sdk-core/libaws-cpp-sdk-core.a ../testing-resources/libtesting-resources.a ../aws-cpp-sdk-core/libaws-cpp-sdk-core.a -lpthread
ld: internal error: atom not found in symbolIndex(__ZN7testing8internal10scoped_ptrINSt3__118basic_stringstreamIcNS2_11char_traitsIcEENS2_9allocatorIcEEEEE5resetEPS8_) for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [aws-cpp-sdk-core-tests/runCoreUnitTests] Error 1
make[2]: *** [aws-cpp-sdk-core-tests/CMakeFiles/runCoreUnitTests.dir/all] Error 2
make[1]: *** [all] Error 2

HOMEBREW_VERSION: 0.9.9
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 2b9a2833bc3c6bc8e7b7344e8178ce98e29ebe4b
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 51f948dc11b2380a4c9211d0cec8d16726a6739a
Core tap last commit: 6 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit haswell
Homebrew Ruby: 2.0.0-p481
GCC-4.2: build 0
Clang: 6.0 build 600
Git: 2.9.2 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.8.0_74
OS X: 10.9.5-x86_64
Xcode: 6.2
CLT: 6.2.0.0.1.1424975374
X11: 2.7.8 => /opt/X11

HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
MAKEFLAGS:
CMAKE_PREFIX_PATH: /usr/local
CMAKE_INCLUDE_PATH: /usr/include/libxml2:/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/usr/local/Library/Homebrew/os/mac/pkgconfig/10.9
HOMEBREW_MAKE_JOBS: 4
ACLOCAL_PATH: /usr/local/share/aclocal
PATH: /usr/local/Library/Homebrew/shims/super:/usr/local/opt/cmake/bin:/usr/local/opt/git/bin:/usr/bin:/bin:/usr/sbin:/sbin

So here's the new error:
```
In file included from /Users/osquery/jenkins/workspace/osqueryMasterBuildOSX10.9/osquery/extensions/interface.cpp:16:
In file included from /Users/osquery/jenkins/workspace/osqueryMasterBuildOSX10.9/osquery/extensions/interface.h:14:
In file included from /Users/osquery/jenkins/workspace/osqueryMasterBuildOSX10.9/include/osquery/extensions.h:15:
In file included from /Users/osquery/jenkins/workspace/osqueryMasterBuildOSX10.9/include/osquery/sql.h:19:
/Users/osquery/jenkins/workspace/osqueryMasterBuildOSX10.9/include/osquery/tables.h:625:57: error: chosen constructor is explicit in copy-initialization
  virtual ColumnAliasSet columnAliases() const { return {}; }
                                                        ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/map:838:14: note: constructor declared here
    explicit map(const key_compare& __comp = key_compare())
             ^
1 error generated.
make[3]: *** [osquery/extensions/CMakeFiles/osquery_extensions.dir/interface.cpp.o] Error 1
make[3]: *** Waiting for unfinished jobs....
In file included from /Users/osquery/jenkins/workspace/osqueryMasterBuildOSX10.9/osquery/extensions/extensions.cpp:19:
In file included from /Users/osquery/jenkins/workspace/osqueryMasterBuildOSX10.9/include/osquery/sql.h:19:
/Users/osquery/jenkins/workspace/osqueryMasterBuildOSX10.9/include/osquery/tables.h:625:57: error: chosen constructor is explicit in copy-initialization
  virtual ColumnAliasSet columnAliases() const { return {}; }
                                                        ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/map:838:14: note: constructor declared here
    explicit map(const key_compare& __comp = key_compare())
             ^
1 error generated.
make[3]: *** [osquery/extensions/CMakeFiles/osquery_extensions.dir/extensions.cpp.o] Error 1
make[2]: *** [osquery/extensions/CMakeFiles/osquery_extensions.dir/all] Error 2
```

Should be fixable.

--- a/include/osquery/tables.h
+++ b/include/osquery/tables.h
@@ -622,7 +622,7 @@ class TablePlugin : public Plugin {
   virtual TableColumns columns() const { return TableColumns(); }
 
   /// Define a map of target columns to optional aliases.
-  virtual ColumnAliasSet columnAliases() const { return {}; }
+  virtual ColumnAliasSet columnAliases() const { return ColumnAliasSet(); }
 
   /**
    * @brief Generate a complete table representation.

